### ISSUE TITLE
Deprecation warning

### ISSUE THREAD

**Issue by bvogel** [leikind/wice_grid#301]
Hi, I'm seeing a lot of these as of late:

```
DEPRECATION WARNING: num_pages is deprecated and will be removed in Kaminari 1.0. Please use total_pages instead.
```

I checked my code and I don't use `num_pages` anywhere, so I suspect wice_grid as culpable.

UPDATE: it seems to be here https://github.com/leikind/wice_grid/blob/rails3/lib/wice/helpers/wice_grid_view_helpers.rb#L688

**Comment by leikind** [comment]
Yes, you are correct

**Comment by bvogel** [comment]
fixed by #302
